### PR TITLE
Trial division algo added

### PR DESCRIPTION
**Pull request by Tejaswika**

### CODE REVIEW

**Review by aashutosh-agrawal — CHANGES_REQUESTED** [review]
Please make the following changes:
1. Remove the template and only keep the TrialDivision function (make it return a bool value)
2. Rename the function to isPrime (You can mention the algorithm name in a comment)
3. Remove the unnecessary comments, (so that it can be easily used as a snippet without much clutter)

**Comment by Tejaswika** [comment]
Done Sir!!! now is it okay??

File: Trial division Algorithm for Prime Factorization.txt [hunk 1]
```diff
@@ -1,50 +1,14 @@
-// CPP implementation of 
+
 // Trial Division Algorithm 
 #include <bits/stdc++.h>
```
Comment (aashutosh-agrawal) [inline Trial division Algorithm for Prime Factorization.txt]: remove this as well.

File: Trial division Algorithm for Prime Factorization.txt [hunk 2]
```diff
@@ -1,50 +1,14 @@
...
-// a prime number or not 
-int TrialDivision(int N){ 
-  
-    // Initializing with the value 2 
-    // from where the number is checked 
-    int i = 2; 
-  
-    // Computing the square root of 
-    // the number N 
+using namespace std;
```
Comment (aashutosh-agrawal) [inline Trial division Algorithm for Prime Factorization.txt]: and this one too.

File: Trial division Algorithm for Prime Factorization.txt [hunk 3]
```diff
@@ -1,50 +1,14 @@
...
-int TrialDivision(int N){ 
-  
-    // Initializing with the value 2 
-    // from where the number is checked 
-    int i = 2; 
-  
-    // Computing the square root of 
-    // the number N 
+using namespace std;  
+int isPrime(int N){
```
Comment (aashutosh-agrawal) [inline Trial division Algorithm for Prime Factorization.txt]: change its return value to bool

File: Trial division Algorithm for Prime Factorization.txt (hunk 1 above)
Comment (aashutosh-agrawal) [reply Trial division Algorithm for Prime Factorization.txt]: Remove the include also, as the code which will use this snippet will already have one "include" line.

**Comment by aashutosh-agrawal** [comment]
Well, Thank you for the changes. Welcome aboard.
I am merging this PR please fill the [merged PR](https://docs.google.com/forms/d/e/1FAIpQLSexn0dft5QJtLvyGr7ME4-cmbYFCZW0jLNQ9bAIaQXF_YFoKQ/viewform) form So that we may count this for the contest.

**Pull request merged**